packages/posthog/src/definitions.ts

```diff
@@ -18,6 +18,34 @@ declare module '@capacitor/cli' {
        * @example 'https://eu.i.posthog.com'
        */
       host?: string;
+      /**
+       * Whether to enable session recording automatically.
+       *
+       * @since 8.0.0
```

Comment:
```suggestion
       * @since 7.3.0
```

We can publish this as minor update. Please also update the other version properties.